codecov.yml

```diff
@@ -1,10 +0,0 @@
-coverage:
```

Comment:
ghhhmmm i needs that file buddy :)

Reply:
very strange . . . I'm not sure how that happenned but got it back in there